components/brave_rewards/resources/extension/brave_rewards/background/reducers/rewards_panel_reducer.ts

```diff
@@ -70,8 +132,19 @@ export const rewardsPanelReducer = (state: RewardsExtension.State | undefined, a
       ) {
         break
       }
-
-      chrome.braveRewards.getPublisherData(tab.windowId, tab.url, tab.favIconUrl || '')
+      let publisherBlob = ''
+      if (tab && tab.url && tab.url.includes('www.twitch.tv')) {
```

Comment:
this check would match on `https://www.twitch.tv.myevildomain.com`. Please either use `new window.URL('...')` to construct a URL object and get the origin there or verify the entire URL prefix.

Reply:
Fixed.